src/data_validation/validation.py

```diff
@@ -0,0 +1,73 @@
+import os
+import toml
+
+from src.utils.hdfs_mods import read_hdfs_csv as read_data
+
+datafilepath = "/ons/rdbe_dev/Frozen_Group_Data2021_244_Headers.csv"
+
+
+def load_schema(filePath: str = "./config/DataSchema.toml") -> dict:
+    """Load the data schema from toml file into a dictionary
+
+    Keyword Arguments:
+        filePath -- Path to data schema toml file
+        (default: {"./config/DataSchema.toml"})
+
+    Returns:
+        A dict: dictionary containing parsed schema toml file
+    """
+    file_exists = os.path.exists(filePath)
+
+    # Check if DataSchema.toml exists
+    if not file_exists:
+        return file_exists
```

Comment:
It isn't. I have swapped the order round for readability now though so read:

if file_exists:
    <load toml file to dictionary>
else:
    return file_exists

i.e. load toml file into a dictionary if the file exists, otherwise return False.